code/modules/atmospherics/machinery.dm

```diff
@@ -7,15 +7,14 @@ ABSTRACT_TYPE(/obj/machinery/atmospherics)
  * Note: A single pipe_network represents a completely open space.
  *
```

Comment:
Could probably ditch this empty block comment line if you like.

Reply:
okee